app/services/journals/create_service.rb

```diff
@@ -874,7 +870,7 @@ def normalize_newlines_sql(column)
 
     def cause_sql(cause)
       # Using the same encoder mechanism that ActiveRecord uses for json/jsonb columns
-      ActiveSupport::JSON.encode(cause || {})
+      ActiveSupport::JSON.encode(cause&.to_h || {})
```

Comment:
Using the `to_hash` method on the cause class, this can be written as:

```suggestion
      ActiveSupport::JSON.encode(cause || {})
```